beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -46,23 +68,24 @@ const (
 // successful. The beacon node is expected to integrate the new block into its state, and
 // therefore validate the block internally, however blocks which fail the validation are still
 // broadcast but a different status code is returned (202). Pre-Bellatrix, this endpoint will accept
-// a `SignedBeaconBlock`. The broadcast behaviour may be adjusted via the `broadcast_validation`
+// a `SignedBeaconBlock`. After Deneb, this additionally instructs the beacon node to broadcast all given signed blobs.
+// The broadcast behaviour may be adjusted via the `broadcast_validation`
 // query parameter.
-func (bs *Server) PublishBlindedBlockV2(w http.ResponseWriter, r *http.Request) {
+func (s *Server) PublishBlindedBlockV2(w http.ResponseWriter, r *http.Request) {
 	ctx, span := trace.StartSpan(r.Context(), "beacon.PublishBlindedBlockV2")
 	defer span.End()
-	if shared.IsSyncing(r.Context(), w, bs.SyncChecker, bs.HeadFetcher, bs.TimeFetcher, bs.OptimisticModeFetcher) {
+	if shared.IsSyncing(r.Context(), w, s.SyncChecker, s.HeadFetcher, s.TimeFetcher, s.OptimisticModeFetcher) {
 		return
 	}
 	isSSZ, err := http2.SszRequested(r)
 	if isSSZ && err == nil {
-		publishBlindedBlockV2SSZ(ctx, bs, w, r)
+		publishBlindedBlockSSZ(ctx, s, w, r, true)
 	} else {
-		publishBlindedBlockV2(ctx, bs, w, r)
+		publishBlindedBlock(ctx, s, w, r, true)
 	}
 }
 
-func publishBlindedBlockV2SSZ(ctx context.Context, bs *Server, w http.ResponseWriter, r *http.Request) {
+func publishBlindedBlockSSZ(ctx context.Context, s *Server, w http.ResponseWriter, r *http.Request, v2 bool) {
```

Comment:
- Can you refactor this to make `*Server` the receiver - `func (s *Server)`? Similarly in other functions.
- The `v2` parameter is not needed. `validateBroadcast` will ignore the request if it doesn't have the appropriate header set.